api/api.js

```diff
@@ -188,6 +188,31 @@ export default (router) => {
             await NinaProcessor.addCollectorForRelease(releasePublicKey, accountPublicKey)
           }
         }
+      } else if (releasePublicKey) {
+        await NinaProcessor.init();
+        const release = await NinaProcessor.program.account.release.fetch(new anchor.web3.PublicKey(releasePublicKey))
+        if (release) {
+          let tokenAccountsForRelease = await NinaProcessor.tokenIndexProvider.connection.getParsedProgramAccounts(
+            new anchor.web3.PublicKey("TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA"), {
+            commitment: NinaProcessor.provider.connection.commitment,
+            filters: [{
+                dataSize: 165
```

Comment:
I'm sure there's a good reason, but what is 165 for? Can we define this as a context to help provide more context as to what this value is intended for?

Reply:
this is the archaic side of solana development (which has maybe gotten better since i first started using this pattern) - explained in depth here: https://solanacookbook.com/guides/get-program-accounts.html#filters

tldr you define which program you want to search for and the size of account and offset with in the account.  the token program doesnt know that a token account is 165 bytes and that the mint parameter being at offset 0